### ISSUE TITLE
Container Logs font settings

### ISSUE THREAD

**Issue by atatarn** [iongion/container-desktop#168]
Could anything be done to the fonts in Container Logs tab? Its pretty much insane right now...
![image](https://github.com/user-attachments/assets/41263dcd-a101-485a-ba1c-7da56818a7f0)

Some controls to change font type and size will be definitely appreciated.

And thanks for this piece of software, it actually does help a lot!

**Comment by iongion** [comment]
Yes, sure, makes sense to have that customizable instead of the default OS monospace, I will need to create some UI for it.
Meanwhile, do you have a favorite font family ?

**Comment by atatarn** [comment]
Consolas is fine, for me  personally an option to control font size & weight is more important than the family. ;)

**Comment by iongion** [comment]
Sure sure, but I need to buy myself some time 😅 - will add it

**Comment by atatarn** [comment]
Thanks! xD

**Comment by iongion** [comment]
Default fonts are now in this PR - https://github.com/iongion/container-desktop/pull/169/files#diff-0c26948eeca1912c6b4a24a7e4f9720a7f3c68f94d2c273aa92585114b187810R3
I've been working on it for quite some time as it includes some advanced security topics and visual improvements. As soon as that PR gets merged, will start working on generic custom settings UI that will cover the custom font family too.

**Comment by iongion** [comment]
Solved in [#5.3.0](https://github.com/iongion/container-desktop/releases/tag/5.3.0)
